src/main/java/racingCar/RacingCar.java

```diff
@@ -0,0 +1,175 @@
+package racingCar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class RacingCar {
+	public static final String ENTER_CAR_NAMES = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).";
+	public static final String WHAT_TIMES = "시도할 횟수는 몇회인가요?";
+	public static final String NAME_ERROR = "[ERROR] 이름이 공백이거나 6자 이상이면 안됩니다. 다시 입력해주세요.";
+	public static final String TIMES_ERROR = "[ERROR] 시도 횟수는 양수인 정수여야 합니다. 다시 입력해주세요.";
+	public static final String RESULT = "실행결과";
+	public static final String COMMA_SPACE = ", ";
+	public static final String FINAL_WINNER = "가 최종 우승했습니다.";
+	public static final String COMMA = ",";
+	public static final int NAME_MAX_LENGTH = 5;
+	public static final int ZERO_TO_NINE_PROBABILITY = 4;
+	private ArrayList<Car> cars = new ArrayList<>();
+	private int maxPosition = Integer.MIN_VALUE;
+	private int times;
+
+	public void start() {
+		List<Car> winners;
+		String[] carNames = racingCarNames();
+		racingCarTimes();
+		makeCars(carNames);
+		race(times);
+		winners = findWinner(cars);
+		printWinners(winners);
+	}
+
+	private String[] racingCarNames() {
+		String input = userInput(ENTER_CAR_NAMES);
+		try {
+			checkEndBlank(input);
+		} catch (RuntimeException e) {
+			return racingCarNames();
+		}
+		return makeCarNames(input);
+	}
+
+	private void checkEndBlank(String input) throws RuntimeException{
+		if (input.endsWith(",")) {
+			System.out.println(NAME_ERROR);
+			throw new RuntimeException();
+		}
+	}
+
+	private void racingCarTimes() {
+		String inputRacingTimes = userInput(WHAT_TIMES);
+		enterTimes(inputRacingTimes);
+	}
+
+	private String userInput(String message) {
```

Comment:
함수는 동사형으로 시작하는 편이 좋아요. 왜냐하면 실제로 **행동**을 하기 때문이죠!! 

입력을 받는 행위를 하는 함수네요. 실제 하는 행위를 반영하여 수정해볼까요?

---
추가로 입력을 받는 행위만 한다고 생각했지만 실제로는 `System.out.println(message)`를 통해 메시지를 출력하기까지 하네요! 이런 경우는 해당 메서드의 이름이 하는 행동을 충분하게 반영하지 못함을 의미합니다!

Reply:
getInput으로 받는 인자 없이, 메서드 이름을 수정하고 System.out.println을 분리했어요. 원래는 2군데 중복된 코드(자동차들 이름, 횟수)를 합친건데 두가지 역할을 하네요.! 그러면 
```suggestion
	private String getInput(String message) {
	    noticeMessage(message);
            Scanner scanner = new Scanner(System.in);
            return scanner.nextLine();
	}
	
	private void noticeMessage(String message) {
	    System.out.println(message);
	}
```
이건 낭비겠죠?

Reply:
이름을 받는 행위와 횟수를 받는 행위를 수행함에 있어 중복된 코드를 제거하시기위해 이렇게 처리하셨군요..!

제가 캐치를 못했네요 😂 

굳이 `noticeMessage`로 안 빼도 될 것 같습니다. 다만, 함수의 이름은 동사로 시작한다는 컨벤션을 기억하면 좋을 것 같아요!

Reply:
넵 함수이름은 동사로 시작 컨벤션 지키겠습니다.